app/presenters/concerns/withdraw_or_decline_application_api_data.rb

```diff
@@ -0,0 +1,9 @@
+module WithdrawOrDeclineApplicationAPIData
+  def schema
+    super.deep_merge!({
+      attributes: {
+        withdrawn_or_declined_for_candidate_by_provider: application_choice.withdrawn_or_declined_for_candidate_by_provider,
```

Comment:
Could we surface to providers the field names `withdrawn_or_declined_for_candidate` as set in the documentation?

Reply:
So I realised we have a withdrawal object with a timestamp, could we update the docs to remove `withdrawn_at` which we added to the draft spec and point to the withdrawal object, and update the specs please?

Reply:
updated spec